db/data/regions/SE/en.yml

```diff
@@ -8,13 +8,15 @@ en:
             default: Street and house number
             optional: Street and house number (optional)
           errors:
-            blank: Enter a street and house number
+            blank_instructional: Enter a street and house number
+            blank_informative: Street and house number is missing
         address2:
           label:
             default: Apartment, suite, etc.
             optional: Apartment, suite, etc. (optional)
           errors:
-            blank: Enter an apartment, suite, etc.
+            blank_instructional: Enter an apartment, suite, etc.
+            blank_informative: Apartment, suite, etc. is missing
```

Comment:
Maybe `Apartment or suite is missing` or `Address line 2 is missing`? cc @ksvihar

Reply:
Updated it to read: `Apartment or suite is missing`